src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java

```diff
@@ -251,7 +225,11 @@ public void visit(PlainSelect plainSelect) {
 
         if (plainSelect.getWhere() != null) {
             buffer.append(" WHERE ");
+            int len = buffer.length();
```

Comment:
Please remove those specific parts from the PR as it does not fit into the generic Deparser.